### PR TITLE
async parallel image upload

### PR DESCRIPTION
**Pull request by kherinwork**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,10 +1,19 @@
+const Sharinpix = require('sharinpix');
+// const secretUrl = 'sharinpix://55a77236-7167-455f-a146-ce2282906880:_EiLHGQM30s_TDPBr58cqA_cOM04yjfFitDnbB0tFLLyroU@kherin.ngrok.io/api/v1';
```
Comment (azharbeebeejaun) [inline index.js]: SECRET URL ! :O :O :O

File: index.js [hunk 2]
```diff
@@ -107,56 +118,22 @@ app.post('/upload-csv', function(req, res) {
     
     let bufferStream = new Stream.PassThrough();
     bufferStream.end(csvFile.data);
-    let csvJsonId = uuidV4();
-    let outputFilePath = `${__dirname}/csv-jsons/${csvJsonId}.csv`;
+    // let csvJsonId = uuidV4();
+    let utcdate = ("0" + date.getUTCDate()).slice(-2);
+    let month =   ("0" + date.getMonth()).slice(-2);
```
Comment (azharbeebeejaun) [inline index.js]: espaces en plus

File: index.js [hunk 3]
```diff
@@ -107,56 +118,22 @@ app.post('/upload-csv', function(req, res) {
...
+    let csvJsonId = date.getFullYear()+'-'+month+'-'+utcdate+'-'+date.getHours()+date.getMinutes()+date.getSeconds();
```
Comment (azharbeebeejaun) [inline index.js]: manque d'espaces

**Review by azharbeebeejaun — CHANGES_REQUESTED** [review]

File: lib/csv-upload.js [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+                                    callback(null, { id: res.id, image_id: image.externalId });
+                                },
+                                (err) => {
+                                    callback(body, null);
+
+                                });
+                            },
+                            function createBox(results, callback){
+                                _.each(results, function(imageImport){
+                                    setTimeout(function() {
```
Comment (kevanmoothien) [inline lib/csv-upload.js]: setTimeout fires only once!!!

**Review by kevanmoothien — CHANGES_REQUESTED** [review]

**Pull request closed without merging**